segway/_util.py

```diff
@@ -55,7 +55,7 @@
 
 DELIMITER_BED = '\t'  # whitespace or tab is allowed
 
-DTYPE_IDENTIFY = intc
+DTYPE_IDENTIFY = 'U16'
```

Comment:
Why are there single quotes on this type and not on the others (`intc`)?

Reply:
`intc` is a numpy-defined type for signed integers compitable with C integers. `'U16'` is an array typestring, which specifies the datatype is up to 16 unicode characters. I haven't found a way to specify this without a typestring, but will add a comment explaining it.

Reply:
To be clear - U16 is a 16 bit unicode type, not 16 unicode characters long? If it's a 16 bit unicode type - this should probably be a UTF-8 equivalent type instead since that's why Python defaults to anyway.

Reply:
`'U16'` is 16 unicode characters long. In typestrings for other datatypes the number denotes the number of bytes for that datatype, but unicode is an exception.